app/src/main/java/sun/trainingcourse/demomvp/adapter/CakeAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package sun.trainingcourse.demomvp.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import sun.trainingcourse.demomvp.R
+import sun.trainingcourse.demomvp.data.Cake
+
+class CakeAdapter(private val cakeList: MutableList<Cake>) :
```

Comment:
Khởi tạo ban đầu Adapter với 1 list rỗng 
```suggestion
class CakeAdapter :
```